### PR TITLE
Adding new guides

### PR DESCRIPTION
**Pull request by s-laugh**
New guides that will help with #1:

- TFVC to Git
- TFS to GCcode

### CODE REVIEW

**Comment by s-laugh** [comment]
> But into our current state. Would we propose a approach for .NET developer by example.

Our current state allows us to write a custom script to inject secrets (environment variables) into a `web.config` as part of the build steps. We don't currently have a way to manage secret files, unless you are using Jenkins.

> I talked with Pierre Lupien last day and he told me something about a DAT file that you have on your secret sharepoint. THen on the the build script, it replace the value from your .DAT entry to your config file.

We actually generate the `.dat` file in the build.

> Maybe I can create a dedicated MD file with a small guide on how to implement the secret replacing step.

A better solution might be to create (or find) a generic script to take a list of variables and inject (replace) them in a file. That way we can share that script with everyone.

File: guides/tfs-to-gccode.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+1. Build your pipeline
+
+## FAQ
+
+### How do you manage TFS related items like UserStory, WorkItem, Bugs, etc, in GitLab?
+
+> It looks very similar in GitLab.
+
+**Labels & Issue Templates!!!**  
+_Labels_ are critical to organizing _Issues_ (or _WorkItems_). _Bugs_ are given a `bug` label. Things like _UserStory_ is given an `enhancement` or `feature request` label.
```
Comment (PhilippeTurcotteGov) [inline guides/tfs-to-gccode.md]:
```suggestion
_Labels_ are critical to organizing _Issues_ (or _WorkItems_). _Bugs_ are given a `bug` label. Things like _UserStory_ are given an `enhancement` or `feature request` label.  
```

File: guides/tfs-to-gccode.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## FAQ
+
+### How do you manage TFS related items like UserStory, WorkItem, Bugs, etc, in GitLab?
+
+> It looks very similar in GitLab.
+
+**Labels & Issue Templates!!!**  
+_Labels_ are critical to organizing _Issues_ (or _WorkItems_). _Bugs_ are given a `bug` label. Things like _UserStory_ is given an `enhancement` or `feature request` label.  
+_Issue Templates_ helps users creating new _Issues_ fill out all the necessary details for a standard _Issue_.
```
Comment (PhilippeTurcotteGov) [inline guides/tfs-to-gccode.md]:
```suggestion
_Issue Templates_ help users creating new _Issues_ to fill out all of the necessary details for a standard _Issue_.
```

File: guides/tfs-to-gccode.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+(Don't move closed items.)
+
+### What is the recommended approach for managing secrets in code?
+
+> When migrating to an open source platform (like GitHub or GCcode) we can't keep hardcoding secrets, and they need to be removed.
+> How should a new developer get the necessary information to input into the web.config? (i.e. : inserting secrets, server name, username, etc)
+
+Whatever tool is used to building your solution for deployment is also responsible for injecting the secrets into the config.
+Depending on the language and framework you are developing with, they may have different methods to managing the injection of those secrets.
+You may need to custom write a build script to search and replace secrets from your builds secret environment variables.
```
Comment (PhilippeTurcotteGov) [inline guides/tfs-to-gccode.md]:
```suggestion
You may need to custom write a build script to search and replace secrets from your build's secret environment variables.
```

File: guides/tfvc-to-git.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+### Is it possible to migrate the code changeset history from TFSV to Git?
+
+> Did you succeeded?
+> Should we do that?
+
+It is possible, and we have successfully done it a few times.
+You can use the [GitTFS tool](https://github.com/git-tfs/git-tfs/blob/master/doc/usecases/migrate_tfs_to_git.md).
+
+There is no legal requirements for us to keep source code history, therefor we do not recommend keeping any history in the conversion.
```
Comment (PhilippeTurcotteGov) [inline guides/tfvc-to-git.md]:
```suggestion
There is no legal requirements for us to keep source code history, therefore we do not recommend keeping any history in the conversion.
```

File: guides/tfvc-to-git.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+> I saw multiples interesting functionalities like the ReadMe.md (i.e.: for the file project configuration) and the Wiki section in markdown.  
+> Personally, I see a lot of pros to use GitLab.
+> Centralize everything, any docs at the same place.
+
+**Absolutely!!**
+
+All documentation should be, at a minimum, moved into source control.
+This allows you to version your documentation with the code it's documenting.
+Taking it a step further to change your documentation to markdown is even better.
+That will alow you to track changes though your documentation, and alow you to begin to automate the documentation.
```
Comment (PhilippeTurcotteGov) [inline guides/tfvc-to-git.md]:
```suggestion
That will allow you to track changes through your documentation, and allow you to begin to automate the documentation.
```

File: guides/tfvc-to-git.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+Wiki's should be used to document parts of your application that don't (are very unlikely) change from version to version.
```
Comment (PhilippeTurcotteGov) [inline guides/tfvc-to-git.md]:
```suggestion
Wiki's should be used to document parts of your application that don't (are very unlikely to) change from version to version.
```

**Pull request merged**